core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/keymanager/delete/KeystoreFileManager.java

```diff
@@ -55,6 +60,39 @@ public void deleteKeystoreFiles(final String pubkey) throws IOException {
     }
   }
 
+  /**
+   * Create Keystore metadata, json and password files
+   *
+   * @param fileNameWithoutExtension File name, usually public key, without extension.
+   * @param jsonKeystoreData Keystore Json data which will be written to
+   *     fileNameWithoutExtension.json
+   * @param password password char[] which will be written as pubkey.password. All the values in the
```

Comment:
s/pubkey.password/fileNameWithoutExtension.password ?

Reply:
good catch ... I also realized I missed the javadoc wording after refactoring it.